.github/workflows/chromatic.yml

```diff
@@ -0,0 +1,32 @@
+# Chromatic action from https://www.chromatic.com/docs/github-actions
+# Using this github action from https://github.com/lightspeedretail/actions-label-prs
+
+name: "chromatic"
+
+on:
+  pull_request:
+    types: [labeled]
+  pull_request_review:
+    types: [submitted]
+  push:
+  workflow_dispatch:
+
+jobs:
+  deploy-storybook:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+      - name: Use Node.js ${{ matrix.node-version }}
+        uses: actions/setup-node@v3
+        with:
+          node-version: ${{ matrix.node-version }}
+          cache: 'npm'
+      - run: npm install
+        env:
+          NODE_AUTH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Publish to Chromatic
+        run: npx chromatic --project-token ${{ vars.CHROMATIC_PROJECT_TOKEN }} -s storybook/public
```

Comment:
do you have this token?

Reply:
${{ vars.CHROMATIC_PROJECT_TOKEN }} used my token for testing. 
You can get the token by joining chromatic and creating a project.

${{ secrets.GITHUB_TOKEN }} is not required.

Reply:
Okay, we can create a project, and add the token to the repository. Anything else to do from our side here?

Reply:
<img width="843" alt="Screenshot 2023-01-31 at 16 22 05" src="https://user-images.githubusercontent.com/27995606/216055496-3c2395fd-2c62-4198-a85c-61a961240d9e.png">

I requested access. The request goes to the owners of the group. @ckifer @arcthur

Reply:
<img width="513" alt="Screenshot 2023-02-01 at 14 26 13" src="https://user-images.githubusercontent.com/27995606/216055655-d70ba03c-548c-4c97-8baa-2a71974c4960.png">

Reply:
Looks like we're good. I added the token to secrets though, so we should reference it like the GH token. Instead of as a var

Reply:
Changed vars to secrets